config/music.json

```diff
@@ -0,0 +1,52 @@
+[
+	{
+		"title":"Endless Space",
+		"artist":"Solus",
+		"license":"/datum/license/ccby3",
+		"audio_file":"sound/ambience/title0.mp3",
+		"url":"https://www.newgrounds.com/audio/listen/74946",
+		"jukebox":1,
+		"roundend":1,
+		"lobbymusic":1
+	},
+	{
+		"title":"Flip-Flap",
+		"artist":"X-CEED",
+		"audio_file":"sound/ambience/title1.mp3",
+		"url":"https://aminet.net/package/mods/xceed/Flipflap",
+		"license":"/datum/license/grandfathered",
+		"jukebox":1,
+		"lobbymusic":1
+	},
+	{
+		"title":"Robocop Gameboy Title Remix",
+		"artist":"CBoyardee",
+		"audio_file":"sound/ambience/title2.mp3",
+		"url":"https://archive.org/details/robocop_20160928",
+		"license":"/datum/license/grandfathered",
+		"jukebox":1,
+		"lobbymusic":1
+	},
+	{
+		"title":"Tintin on the Moon Remix",
+		"artist":"Cuboos (Original Artist: Jeroen Tel)",
+		"url":"https://www.youtube.com/watch?v=Hz6KO0gnrS8&t=0s",
```

Comment:
This is not the actual link to the remix. This a link to the original (Jeroen Tel) version.

`https://www.youtube.com/watch?v=YKVmXn-Gv0M` is the correct youtube link for our title3 sound file.

Reply:
Good catch!

Reply:
I wasn't quite sure how the licensing and linking should work on this